Author: Roland Rosenfeld <roland@debian.org>
Description: Change the global default from standard.Cautious to
 standard.Medium, which is similar to the old 3.0.3 behavior and 
 doesn't change too much.

--- a/match-all.action
+++ b/match-all.action
@@ -10,7 +10,16 @@
 +change-x-forwarded-for{block} \
 +client-header-tagger{css-requests} \
 +client-header-tagger{image-requests} \
++deanimate-gifs{last} \
++filter{refresh-tags} \
++filter{img-reorder} \
++filter{banners-by-size} \
++filter{webbugs} \
++filter{jumping-windows} \
++filter{ie-exploits} \
 +hide-from-header{block} \
++hide-referrer{conditional-block} \
++session-cookies-only \
 +set-image-blocker{pattern} \
 }
 / # Match all URLs
